examples/configs/BUILD.bazel

```diff
@@ -0,0 +1,4 @@
+filegroup(
+    name = "configs",
+    srcs = glob(["*"]),
+)
```

Comment:
Once there is a separate example for zero-copy stuff, I'd suggest to keep the config in there as well.